content/600-about/05-faq.mdx

```diff
@@ -8,7 +8,9 @@ metaDescription: 'Frequently asked questions (FAQ) about the Prisma tools and th
 
 Yes, Prisma Client provides two methods called `$queryRaw` and `$executeRaw` which you can use to send raw SQL to the database. Learn more about these in the docs [here](../concepts/components/prisma-client/raw-database-access).
 
-Note that Prisma currently doesn't support full [transactions](../concepts/components/prisma-client/transactions). If your application needs transactions, you can use Prisma Client alongside other lightweight query builder such as [knex.js](https://www.github.com/tgriesser/knex) to perform one-off queries as transaction.
+Note that Prisma currently does not support _long-running transactions_. If your application needs long-running transactions, you can use Prisma Client alongside other lightweight query builder such as [knex.js](https://www.github.com/tgriesser/knex) to perform one-off queries as transaction.
```

Comment:
We should have an issue about that we are working on it, or a roadmap entry we can link to here.

Reply:
Added ✔️